.cirrus.yml

```diff
@@ -24,48 +24,52 @@ env:
     FEDORA_NETAVARK_IMAGE: "fedora-netavark-${IMAGE_SUFFIX}"
 
 
-gcp_credentials: ENCRYPTED[f6a0e4101418bec8180783b208721fc990772817364fed0346f5fd126bf0cfca03738dd8c7fb867944637a1eac7cec37]
-
-
-# Default VM to use unless set or modified by task
-gce_instance: &standard_vm
-    image_project: "libpod-218412"
-    zone: "us-central1-c"
-    cpu: 2
-    memory: "4Gb"
-    disk: 200  # GB, do not set <200 per gcloud warning re: I/O performance
-    image_name: "${FEDORA_NETAVARK_IMAGE}"
```

Comment:
Yeah, I think next step is to remove this, and the top-level `ec2_instance` block.  Define them in separate tasks, and set the YAML anchors there for re-use (if needed).  The top-level definitions are suppose to be "defaults" for when a task doesn't define one.  I have a strong feeling that having TWO defaults is going to really confuse Cirrus-CI.